pkg/metrics/metrics.go

```diff
@@ -241,10 +246,10 @@ var (
 	// ephemeralMetrics defines temporary metrics whose data should be cleared when an upgrade completes
 	ephemeralMetrics = []*prometheus.GaugeVec{
 		metricValidationFailed,
-		metricClusterCheckFailed,
 		metricScalingFailed,
 		metricUpgradeWindowBreached,
 		metricUpgradeControlPlaneTimeout,
+		metricUpgradeoperatorHealthcheckFailed,
```

Comment:
```suggestion
		metricUpgradeHealthCheckFailed,
```
Considering previous suggestion is accepted, this reference would need to change as well